app/com/arpnetworking/utility/DefaultOrganizationProvider.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2018 Smartsheet.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.utility;
```

Comment:
Why is this in a different package than the interface?

Reply:
^ If we follow the existing pattern it should probably be in a `impl` sub-package.